### PR TITLE
Feature/modal confirmation

### PR DESCRIPTION
**Pull request by lplume**
**tl;dr;** this pull cross port snipe-it modal and susbtitute this snipe/laravel4-starter#14

**be careful**
I change some template section (which i added myself in the previous PR), this may broke something if someone used that section in their projects.
I came across cartalyst/sentry/issues#463 , i do not put the logic in the template, but in the user's model.
This last is up to you, we may talk about this.

```php
    public function accountStatus()
    {
        if(is_null($this->deleted_at))
            $throttle = Sentry::findThrottlerByUserId($this->id);
        else
            return;

        if ($throttle->isBanned()) {
            return 'banned';
        } elseif ($throttle->isSuspended()) {
            return 'suspended';
        } else {
            return '';
        }

    }
```

**what i did**
1. remove confirmation route (before i fetch translated modal w/ajax)
2. remove (1) controllers method
3. move the modal into the default backend template
4. split the section (blade) body_bottom into: javascript_bottom (after jquery and bootstrap include), body_bottom (after the footer)
5. set the content of the model through data-attribute

### CODE REVIEW

**Comment by snipe** [comment]
Awesome, thanks!
